docker-compose.yml

```diff
@@ -1,46 +1,91 @@
-version: '3.8'
+name: ffc-api-modifier
```

Comment:
it has been marked as obsolete and it will emit a warning each time. See the [Compose File Reference](https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete).